nixos/modules/services/misc/geoipupdate.nix

```diff
@@ -139,7 +150,40 @@ in
         ReadWritePaths = cfg.settings.DatabaseDirectory;
         RuntimeDirectory = "geoipupdate";
         RuntimeDirectoryMode = 0700;
+        CapabilityBoundingSet = "";
+        PrivateDevices = true;
+        PrivateMounts = true;
+        PrivateUsers = true;
+        ProtectClock = true;
+        ProtectControlGroups = true;
+        ProtectHome = true;
+        ProtectHostname = true;
+        ProtectKernelLogs = true;
+        ProtectKernelModules = true;
+        ProtectKernelTunables = true;
+        ProtectProc = "invisible";
+        ProcSubset = "pid";
+        SystemCallFilter = [ "@system-service" "~@privileged" "~@resources" ];
+        RestrictAddressFamilies = [ "AF_INET" "AF_INET6" ];
+        RestrictRealtime = true;
+        RestrictNamespaces = true;
+        MemoryDenyWriteExecute = true;
+        LockPersonality = true;
+        SystemCallArchitectures = "native";
+        DeviceAllow = "";
+        DevicePolicy = "closed";
+      };
+    };
+
+    systemd.timers.geoipupdate-initial-run = {
```

Comment:
What? Why? The `requires` should pull in that unit anyway

Reply:
Which `requires` are you referring to? The service is only run by a timer. Adding it to `multi-user.target` makes it run on every configuration switch, which is probably too often.